src/interfaces/alteration-interface.ts

```diff
@@ -0,0 +1,12 @@
+import { EntityTypes } from '@/enums'
+
+/** A filing's alteration object from the API. */
+export interface AlterationIF {
+  alterCorpType?: {
+    corpType: EntityTypes;
+  }
+  alterResolutions?: any;
```

Comment:
There is no sub structure like alterResolutions. We have an array of resolution dates within alterShareStructure

Reply:
Fixed. Please review latest code.